orchestra/workflow/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class WorkflowError(Exception):
```

Comment:
I'll move this to core.errors for consistency now, and we can discuss later--I don't feel strongly one way or the other.